src/background.c

```diff
@@ -88,6 +88,11 @@ NONMATCH("asm/non_matching/engine/sub_8002B20.inc", bool32 sub_8002B20(void))
     s32 j;
     u16 k;
 
+#if (RENDERER == RENDERER_OPENGL)
```

Comment:
Might as well be consistent and do `if (RENDERER == RENDERER_SOFTWARE)` around the whole thing?

Reply:
For such a big function? I'd rather keep it with a short `#ifdef`.